pkgs/by-name/me/mealie/package.nix

```diff
@@ -41,22 +41,20 @@ let
   };
 in
 
-pythonpkgs.buildPythonPackage rec {
+pythonpkgs.buildPythonApplication rec {
   pname = "mealie";
   inherit version src;
   pyproject = true;
 
-  nativeBuildInputs = [
-    pythonpkgs.poetry-core
-    makeWrapper
-  ];
+  build-system = with pythonpkgs; [ poetry-core ];
+
+  nativeBuildInputs = [ makeWrapper ];
```

Comment:
This is surprising - both of these attributes' items are concatenated?

Reply:
Yes